ThirdPartyNotices.txt

```diff
@@ -14,7 +14,9 @@ The attached notices are provided for information only.
 1. android/platform/tools/base (https://android.googlesource.com/platform/tools/base/+/d41d662dbf89f9b60ca6256415a059c0107749b8/sdk-common/NOTICE)
 2. bazelbuild/bazel (https://github.com/bazelbuild/bazel/)
 3. force-net/crc32.net (https://github.com/force-net/Crc32.NET)
-4. nunit/nunitlite (https://github.com/nunit/nunitlite/)
+4. google/bionic (https://android.googlesource.com/platform/bionic/)
+5. nunit/nunitlite (https://github.com/nunit/nunitlite/)
+6. tessil/robin-map (https://github.com/Tessil/robin-map)
```

Comment:
This seems odd: the in-repro `ThirdPartyNotices.txt` only needs to contain stuff *within this repo*, e.g. files in `src-ThirdParty`.  A *git submodule* doesn't need to be listed in this file; instead, it needs to be listed in the *generated* `bin/$(Configuration)/lib/xamarin.android/ThirdPartyNotices.txt` file.